airflow-core/docs/img/airflow_erd.sha256

```diff
@@ -1 +1 @@
-4cda55eb221ee0340749c8dd41af7603220d7b300e2e808d733239a86e0c2837
\ No newline at end of file
+80d87cc89c29fd73466a8625f82dfcbff347e9e72fee498bcb61a0acb90bf3a1
```

Comment:
hmm, bad conflict resolution? Don't think the other changes should change this.

Reply:
does it perhaps regenerate the hash whenever you change the migration files?  if the static check didn't fail, it must be right